src/app/src/data/disadvantagedTracts.json

```diff
@@ -0,0 +1,338 @@
+[
```

Comment:
I transformed the original CSV with a pivot table (in Apple Numbers) and re-exported to CSV. I used python to dump it to JSON, and find and replace to change TRUE and FALSE to NotDisadvantaged and Disadvantaged, etc.